test/validation/org/apache/cassandra/bridges/ccmbridge/CCMBridge.java

```diff
@@ -35,8 +35,6 @@
 
     private int nodeCount;
     static final File CASSANDRA_DIR = new File("./");
-
-    private final Runtime runtime = Runtime.getRuntime();
```

Comment:
No objection to this change. The other bridges will most likely re-implement this, so smart to move it to the base class.